### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by luc-demeyer**
fix typo : gorups -> groups

### CODE REVIEW

File: account_mtd/security/groups.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
     <data>
-        <record id="mtd_gorups_category" model="ir.module.category">
+        <record id="mtd_groups_category" model="ir.module.category">
```
Comment (Nick-OpusVL) [inline account_mtd/security/groups.xml:4]: Could you rename its usages too please?

File: account_mtd_vat/security/groups.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
     <data>
-        <record id="mtd_vat_gorups_category" model="ir.module.category">
+        <record id="mtd_vat_groups_category" model="ir.module.category">
```
Comment (Nick-OpusVL) [inline account_mtd_vat/security/groups.xml:4]: Could you rename its usages too please?

**Review by Nick-OpusVL — CHANGES_REQUESTED** [review]

**Comment by luc-demeyer** [comment]
@Nick-OpusVL 
refs now also aligned.

**Comment by Nick-OpusVL** [comment]
@ManishS-OpusVL I'd like to get this contribution rebased, tested and ported across the odoo versions before we give this to the wide world.  Less risk (though risk is low anyway) to apply and test now than when lots of users.

**Comment by Nick-OpusVL** [comment]
Merged manually after rebase and test.  Thanks @luc-demeyer

**Pull request closed without merging**